### ISSUE TITLE
Add method_missing() or other alternatives?

### ISSUE THREAD

**Issue by konung** [opal/opal-jquery#4]
Hello. Just discovered opal - and been playing with it since last  night. Like it a lot - much better than other implementions of this idea that I've seen. Anyways, what would be great  is to somehow implement or fake method missing - this way you can get native functionality  instantaneously to almost any jquery lib such as jquery ui or wijmo, etc.

I was reading around and it seems that there is already native support for it (similar to method_missing - there was a dicussion on Yehuda Katz's blog) in newer version of mozilla, however that doesn't help - cause at least 50 of people still use  IE and according to my largest project - in corp env that's more like 90% plus they tend to use older versions like IE 6 & IE 7

Anyways. I found some suggestions online - but non of them seem robust enough.
Right now when I needed sortable from jquery ui I did this in my app.js.opal ( integrating with rails)
( trying to implement one of the demos from jqueryui site)

in my applicaiton.html.haml file:

``` haml
    :plain
      <ul id="sortable">
          <li class="ui-state-default"><span class="ui-icon ui-icon-arrowthick-2-n-s"></span> Item 1</li>
          <li class="ui-state-default"><span class="ui-icon ui-icon-arrowthick-2-n-s"></span> Item 2</li>
          <li class="ui-state-default"><span class="ui-icon ui-icon-arrowthick-2-n-s"></span> Item 3</li>
          <li class="ui-state-default"><span class="ui-icon ui-icon-arrowthick-2-n-s"></span> Item 4</li>
          <li class="ui-state-default"><span class="ui-icon ui-icon-arrowthick-2-n-s"></span> Item 5</li>
          <li class="ui-state-default"><span class="ui-icon ui-icon-arrowthick-2-n-s"></span> Item 6</li>
          <li class="ui-state-default"><span class="ui-icon ui-icon-arrowthick-2-n-s"></span> Item 7</li>
      </ul>  
```

in my  javascripts/app.js.opal

``` ruby
class Element
  def sortable(opacity="0.5")
    `#{self}.sortable({opacity: #{opacity}})`
  end  
  def hide(duration="slow")
    `#{self}.hide(duration)`
  end  
end  

Document.ready? do
  list = Document['#sortable']
  list.sortable(0.9).hide
end
```

Basically just extending Element class. It's fine I guess but I just miss method_missing :-)  Any thoughts?

P.S.:

Huge props for working chaining!
